docs/lakehouse-delta-sink.md

```diff
@@ -0,0 +1,202 @@
+---
+dockerfile: "https://hub.docker.com/r/streamnative/pulsar-io-lakehouse"
+download: "https://github.com/streamnative/pulsar-io-lakehouse/releases"
+alias: Lakehouse delta Lake Sink Connector
+---
+
+The Lakehouse [Delta Lake](https://delta.io/) sink connector fetches data from a Pulsar topic and saves data to the Lakehouse tables.
+
+![](/docs/lakehouse-sink.png)
+
+## Quick start
+
+### Prerequisites
+
+Tips: Only support use AWS S3 as storage.
+
+The prerequisites for connecting an Lakehouse delta Lake sink connector to external systems include:
+
+1. Create cloud buckets in **AWS S3**.
+2. Create the [AWS User](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users_create.html) and create `AccessKey`(Please record `AccessKey` and `SecretAccessKey`).
+3. Assign permissions to User / ServiceAccount, and ensure they have the following permissions to the AWS S3.
+
+```json
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "s3:ListBucket"
+      ],
+      "Resource": [
+        "{Your bucket arn}"
+      ]
+    },
+    {
+      "Effect": "Allow",
+      "Action": [
+        "s3:PutObject",
+        "s3:GetObject",
+        "s3:DeleteObject"
+      ],
+      "Resource": [
+        "{Your bucket arn}/*"
+      ]
+    }
+  ]
+}
+```
+
+### 1. Create a connector
+
+The following command shows how to use [pulsarctl](https://github.com/streamnative/pulsarctl) to create a `builtin` connector. If you want to create a `non-builtin` connector,
+you need to replace `--sink-type lakehouse-delta-lake` with `--archive /path/to/pulsar-io-lakehouse-cloud.nar`. You can find the button to download the `nar` package at the beginning of the document.
+
+{% callout title="For StreamNative Cloud User" type="note" %}
+If you are a StreamNative Cloud user, you need [set up your environment](https://docs.streamnative.io/docs/connector-setup) first.
```

Comment:
I think this is not the builtin connector? Why don't we give the --archive command as an example?

Reply:
This is a generic template, as some connector are also exposed to non-Streamnative Cloud users.